### PR TITLE
T/51: Underline 

### LINKED ISSUE #51: Underline
Hello,

this question can seem a bit stupid but I was wondering why there is no "underline" button in this package. Is it a specific challenge or were "bold" and "italic" just a test to see if it works?

If it's the later and if I'm able to create an underline command (based on the bold one, it shouldn't be too complicated), is it possible for me to contribute to the project? (Sorry I'm not used to the contributing part of Github).

Thanks

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Introduced the `Underline` plugin. Closes #51.

### CODE REVIEW

**Comment by Reinmar** [comment]
You need to update `contexts.json`.

**Comment by pomek** [comment]
> You need to update `contexts.json`.

Done in e14259d.

Another question related to translation – should I do anything with translations files?

File: src/underline.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const command = editor.commands.get( 'underline' );
+		const keystroke = 'CTRL+U';
```
Comment (Comandeer) [inline src/underline.js]: Yes, there's shortcut for it in CKE4 – `Ctrl+U`. The same shortcut is also available in Word, so it's probably safe to  suppose that it's a _de facto_ standard.

Comment (pomek) [inline src/underline.js]: In Pages `CMD+U` also works as I expected (makes underline).

**Comment by Reinmar** [comment]
> Another question related to translation – should I do anything with translations files?

Nope. Uploading and downloading data to/from Transifex is done by me.

File: src/underlineengine.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module basic-styles/underline
```
Comment (szymonkups) [inline src/underlineengine.js]: `@module basic-styles/underlineengine`

Comment (pomek) [inline src/underlineengine.js]: Fixed in 2a1e748.

**Pull request merged**